### PR TITLE
Preamble questions#148

### LINKED ISSUE #148: Show the Questions in the Candidate Recorder Landing Page
When candidates are referred to the recorder page (from Bria) they want to know what the question are.  

If you go to  https://undebate-qa.herokuapp.com/candidate-conversation-candidate-recorder  -- or it might be schoolboard-conversation-..  if you haven't updated to the latest iota.json that's the first page they see.

Its in app/components/CandidatePreamble

the question that are going to be asked are passed to app/components/webComponents/Undebate as this.props.participants.moderator.agenda

it looks like this:
```
"agenda": [
            [
              "Introductions",
              "1- Who you are",
              "2- Where you are",
              "3- One word to describe yourself",
              "4- What office you are running for"
            ],
            ["What type of skills should students be learning for success in the 21st century?"],
            ["Closing Remarks"]
          ],
```
Also, the last item in the agenda isn't a question for the candidate to answer - it's the moderator getting in the closing statements.

I think it would also be useful to incorporate the time limits, which is also in this.props..participantes.moderator.timeLimits
```
          "timeLimits": [10, 60, 60]
```
So the task is to incorporate the questions and timelimits into the UI design and update these components to implement that.

### PR DESCRIPTION
**Pull request by ddfridley**
The questions for the candidates are added to the preamble, as additional lines of text.  Since the preamble is limited to the screen height, if there are too many questions it will run below what the users can see.  But I tested this with election data and a variety of landscape dimensions and it all fit.  
image
This was done by Dana, I just reduced the font size and tested it. This fixes #148

### CODE REVIEW

**Comment by ddfridley** [comment]
I think this is ready to Squash and merge, but take a look.

**Comment by poornaraob** [comment]
@MrNanosh , @ddfridley In the questions listing in candidate recorder landing page (shared on slack), I found an extra content after second question : 1 Look into camera while listening to example speaker as your placeholder video is recorded" is that needed?
Also, there is numbering 1, 2 added in the display of questions.

File: app/components/candidate-preamble.jsx [hunk 1]
```diff
@@ -77,20 +78,39 @@ class CandidateJoin extends Join {
 }
 
 class CandidatePreamble extends React.Component {
+  makeQuestions = questions => {
```
Comment (luiscmartinez) [inline app/components/candidate-preamble.jsx:81]: This method is hwaatt 🔥🔥🔥
Cool way you handle nested Arrays in this case

File: app/components/candidate-preamble.jsx [hunk 2]
```diff
@@ -26,11 +27,11 @@ const styles = {
       left: '-100vw',
     },
     '& ul': {
-      paddingTop: '0.5em',
-      paddingLeft: '2.5em',
+      paddingTop: '0.5em!important',
```
Comment (luiscmartinez) [inline app/components/candidate-preamble.jsx:30]: I don't recall but why do we have to use important ? 😱👻

**Comment by ddfridley** [comment]
On !important it's because something else was defining the style and over riding this definition so important overrides.  A better fix would be to fix it at the root but that could have side effects such is the nature of css. 


⁣Get BlueMail for Android ​

**Comment by ddfridley** [comment]
I showed this to our contact at "Bruce" and she had some suggestions, but they effect what's in the agenda, not showing the agenda.  So I'm working to talk with her to resolve.

**Comment by MrNanosh** [comment]
@poornaraob the questions include numbers in the array. I thought about removing them but it is better to handle that in the database as for the extra content, is it in the questions array or is the content from the rest of the preamble?

**Comment by MrNanosh** [comment]
@ddfridley what suggestions did your contact give you? do we need to make a new issue?

File: app/components/candidate-preamble.jsx (hunk 1 above)
Comment (MrNanosh) [reply app/components/candidate-preamble.jsx:81]: thanks!

File: app/components/candidate-preamble.jsx (hunk 2 above)
Comment (MrNanosh) [reply app/components/candidate-preamble.jsx:30]: a class could  be used  to remove important as long as whatever selector is used is more specific than the style that overrides it

**Comment by ddfridley** [comment]
The her request amounts to changing the contents of the database.  I 
want to talk it through with her. but it won't be before monday.  If you 
want to pull this into what else you are working on that's fine.  I may 
change the conent of the db, or we may make requests an additional issue.

d.

**Comment by MrNanosh** [comment]
@ddfridley did you open a new branch or did you make your changes to the branch I opened?

**Comment by ddfridley** [comment]
I made changes to the branch you opend - thats here.  If that's a surprise, of not the best way lets discuss.  I was hoping to merge it already - but I need to resolve the concern.

**Comment by ddfridley** [comment]
fixes #148 
Had trouble with css classes in recorder bar button and ended up converting the candidate-preamble component to functional and using createUseStyle() for the styles. 

Worked with BP to agree to this rendering:
image

**Comment by ddfridley** [comment]
It all looks good and can be closed. But I'm having an issue with 
another change, and BP that I'm working on.

**Pull request merged**